tipg/db.py

```diff
@@ -29,7 +31,8 @@ async def connect_to_db(
     """Connect."""
     if not settings:
         settings = PostgresSettings()
-
+    apisettings = APISettings()
```

Comment:
I think it would be better to pass the `functions_directory` as an optional input to the connect_to_db function or as PostgreSettings attribute instead of using the  `APISettings`